lib/openshift/network.rb

```diff
@@ -0,0 +1,14 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # @note represents an OpenShift environment Storage Class
+  class Network < ClusterResource
+    RESOURCE = "networks"
```

Comment:
usually RESOURCE is `io.openshift...` nowadays. You will see this in YAML output of objects.

Reply:
other than that lg